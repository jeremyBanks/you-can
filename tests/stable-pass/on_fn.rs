#![warn(unsafe_code)]

#[you_can::turn_off_the_borrow_checker]
fn main() {
    let mut source = 1;
    let mutable_alias = &mut source;
    let immutable_alias = &source;
    source = 2;
    *mutable_alias = 3;
    println!("{immutable_alias}");
}
