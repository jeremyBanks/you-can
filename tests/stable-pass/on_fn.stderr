
 DANGER   This project is using the the #[you_can::turn_off_the_borrow_checker]
 DANGER   macro, which is inherently unsafe, unsound, and unstable. This is not
 DANGER   suitable for any purpose beyond educational experimentation.

